Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingCoordinatorProxy.cpp

```diff
@@ -132,6 +132,18 @@ void RemoteScrollingCoordinatorProxy::handleWheelEvent(const NativeWebWheelEvent
 #endif
 }
 
+void RemoteScrollingCoordinatorProxy::scrollingTreeNodeWillStartScroll(ScrollingNodeID nodeID)
+{
```

Comment:
We've lost the `webPageProxy().scrollingNodeScrollWillStartScroll(nodeID);` that `remotescrollingcoordinatorproxyios::scrollingTreeNodeWillStartScroll()` had. Doesn't that break things?

Reply:
fixed